src/services/api/actions/getListingInformationByName.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Gets a listing information from the Listing Information API matching the requested pretty URL name
+ *
+ * @param {Object} params the parameters to fetch
```

Comment:
This doesn't seem to be a param in the function. Similar to the issue above.

Reply:
Fixed!